expression/builtin_string.go

```diff
@@ -3216,18 +3225,24 @@ func (b *builtinInsertBinarySig) evalString(row chunk.Row) (string, bool, error)
 	}
 
 	if length > strLength-pos+1 || length < 0 {
+		if uint64(pos-1+int64(len(newstr)))*uint64(mysql.MaxBytesOfCharacter) > b.maxAllowedPacket {
```

Comment:
function `len()` returns the number of bytes the string contains, no need to multiple `mysql.MaxBytesOfCharacter`.